example-esp8266.yaml

```diff
@@ -12,6 +12,8 @@ packages:
     # Change this file to the correct one for your device
     file: econet_heatpump_water_heater.yaml
     # file: econet_hvac.yaml
+    # Use this if you have a comminicating outdoor unit (ODU)
+    # files: [econet_hvac.yaml, econet_hvac_odu.yaml]
```

Comment:
This affects people who do have ODU through right? They will need to comment out `econet_hvac.yaml` and uncomment `files: [econet_hvac.yaml, econet_hvac_odu.yaml]` in order to get the same results?

Probably not many people, but just something to consider :p